crypto/x509/v3_group_ac.c

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright 1999-2021 The OpenSSL Project Authors. All Rights Reserved.
```

Comment:
The year looks wrong here.

Reply:
This is duplicated in most files... Should just be "Copyright 2023 The OpenSSL ..."